c/tskit/trees.h

```diff
@@ -764,15 +764,99 @@ void tsk_tree_print_state(const tsk_tree_t *self, FILE *out);
 @{
 */
 
-/* Returns true if u is a descendant of v; false otherwise */
-bool tsk_tree_is_descendant(const tsk_tree_t *self, tsk_id_t u, tsk_id_t v);
-bool tsk_tree_is_sample(const tsk_tree_t *self, tsk_id_t u);
+/**
+@brief Returns the parent of the specified node.
```

Comment:
The return value is the err code.
```suggestion
@brief Gets the parent of the specified node.
```